guide/onboarding/manual-backup.md

```diff
@@ -11,9 +11,30 @@ main_classes: -no-top-padding
 
```

Comment:
Heading image missing from this page.

Reply:
Same as the other page -- I added in the one from the main Backups > Onboarding page for now.